tests/gpflow/models/test_gpr_posterior.py

```diff
@@ -0,0 +1,50 @@
+from itertools import product
+from typing import Tuple
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+import gpflow
+from gpflow.models.gpr import GP_deprecated, GP_with_posterior
+from gpflow.posteriors import PrecomputeCacheType
+
+input_dim = 7
+output_dim = 1
+
+
+def make_models(regression_data):
+    k = gpflow.kernels.Matern52()
+
+    mold = GP_deprecated(data=regression_data, kernel=k)
+    mnew = GP_with_posterior(data=regression_data, kernel=k)
+    return mold, mnew
+
+
+@pytest.mark.parametrize("cache_type", [PrecomputeCacheType.TENSOR, PrecomputeCacheType.VARIABLE])
+def test_old_vs_new_gp_vs_new_with_posterior(cache_type: PrecomputeCacheType):
+    X = np.random.randn(100, input_dim)
+    Y = np.random.randn(100, output_dim)
+    Xt = tf.convert_to_tensor(X)
```

Comment:
Are these `convert_to_tensor` calls necessary? TensorFlow should be able to do this conversion automatically.

Reply:
No you are correct. I think I was just doing some ignorant/lazy copy/paste from the previous test..now fixed (along with the other test)